### PR TITLE
Add the discovery time to Issue class

### PR DESCRIPTION
**Pull request by norhh**

### CODE REVIEW

File: mythril/start_time.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from time import time
+from mythril.support.support_utils import Singleton
+
+
+class StartTime(metaclass=Singleton):
```
Comment (JoranHonig) [inline mythril/start_time.py]: documentation 😇

Comment (JoranHonig) [reply mythril/start_time.py]: Also I think we should move this to support

File: mythril/mythril.py [hunk 2]
```diff
@@ -36,8 +36,11 @@
 from mythril.support.truffle import analyze_truffle_project
 from mythril.ethereum.interface.leveldb.client import EthLevelDB
 from mythril.laser.smt import SolverStatistics
+from mythril.start_time import StartTime
+
 
 log = logging.getLogger(__name__)
+start_time = StartTime()
```
Comment (JoranHonig) [inline mythril/mythril.py]: I'm thinking that we should maybe re-initialize the start time for each contract that is being analyzed, wdyt?

Comment (norhh) [reply mythril/mythril.py]: what was I thinking when I put it there :man_facepalming: , will move it there :+1:

File: mythril/start_time.py (hunk 1 above)
Comment (norhh) [reply mythril/start_time.py]: Moved

**Pull request merged**